src/plasma/plasma_manager.cc

```diff
@@ -158,7 +161,8 @@ typedef struct {
   /** The object requests for this wait request. Each object request has a
    *  status field which is either PLASMA_QUERY_LOCAL or PLASMA_QUERY_ANYWHERE.
    */
-  ObjectRequest *object_requests;
+  std::unordered_map<ObjectID, ObjectRequest, decltype(&hashObjectID)>
+      *object_requests;
```

Comment:
It wasn't necessary to pass in the third argument when we did this in the plasma store https://github.com/ray-project/ray/pull/324/files#diff-450748b4523710897a16506dabc79950R119, so perhaps it can be avoided here?